<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>350</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans Korean Regular</family>
   </font>
  </property>
  <widget class="QLineEdit" name="nickname_edit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Korean Regular</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Korean Regular</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>용병 닉네임</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Korean Regular</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;용병 티어&lt;span style=&quot; vertical-align:sub;&quot;&gt;(mmr)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Korean Regular</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>290</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>2</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Korean Regular</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mmr_combo_box">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mmr_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>631</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="manual_radio">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>수동 입력</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mmr_edit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Korean Regular</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="alert_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Korean Regular</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
